src/test/java/teammates/test/cases/search/FeedbackResponseCommentSearchTest.java

```diff
@@ -0,0 +1,97 @@
+package teammates.test.cases.search;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.FeedbackResponseCommentSearchResultBundle;
+import teammates.common.datatransfer.attributes.FeedbackResponseCommentAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.storage.api.FeedbackResponseCommentsDb;
+
+/**
+ * SUT: {@link FeedbackResponseCommentsDb},
+ *      {@link teammates.storage.search.FeedbackResponseCommentDocument},
+ *      {@link teammates.storage.search.FeedbackResponseCommentSearchQuery}.
+ */
+public class FeedbackResponseCommentSearchTest extends BaseSearchTest {
```

Comment:
In that case the test may be in the right place. However, this feature needs testing at higher levels too, especially at UI level.